src/parser-parse5.js

```diff
@@ -6,7 +6,8 @@ function parse(text) {
   // Inline the require to avoid loading all the JS if we don't use it
   const parse5 = require("parse5");
   try {
-    const ast = parse5.parse(text, {
+    const isFragment = !/^\s*<(!doctype|html|head|body)/i.test(text);
```

Comment:
It won't actually throw, it will error correct, going from:

```html
<div></div>
```

to

```html
<!DOCTYPE html>
<html>
  <head></head>
  <body>
    <div>
  </body>
</html>
```